plugins/catalog-backend-module-scaffolder-relation-processor/package.json

```diff
@@ -65,7 +66,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/janus-idp/backstage-plugins",
```

Comment:
Try changing it to 
```json
  "repository": "github:janus-idp/backstage-plugins",
  ```
and do the same for the `audit-log-node`?

Reply:
argocd plugin has this more detailed convention:

```
  "repository": {
    "type": "git",
    "url": "https://github.com/janus-idp/backstage-plugins",
    "directory": "plugins/argocd"
  },
```

and it's been working for a while now